radio/src/pulses/pulses_arm.cpp

```diff
@@ -209,6 +209,8 @@ void setupPulses(uint8_t port)
 #endif
     case PROTO_PPM:
       setupPulsesPPM(port);
+      // 45 ms is the worst case length of the PPM frame length (800 µs between channels and 16 channels
```

Comment:
To me that sound still like the mixer runs only every 2nd ppm frame but maybe I just misunderstood that paramter. I will just remove the comment